### ISSUE TITLE
Point two subdomains to same application

### ISSUE THREAD

**Issue by sztosz** [typicode/hotel#80]
I want to point app.something.dev and api.something.dev to be pointing to same application. If i make two config files with different names i get `A server is already running. Check /home/sztosz/app_dir/tmp/pids/server.pid.` Is there a way to serve app under two diffrent subdomains?

**Comment by typicode** [comment]
Hi @sztosz,

Wildcard subdomains have been added in the latest version, so you should be able to have two subdomains pointing to the same app.

For example:

```
my-project$ hotel add 'cmd' -n something
```

You get `something.dev` pointing to `my-project` and also `*.something.dev` pointing to `my-project`.

Does it help?

**Comment by sztosz** [comment]
Not really :) I have complicated domain structure in our company product. We got our landing site at example, we got main product at app.example we got public api at api.example and internal one at internal-api.example and few other ;) 

Anyway our application serves either app or api or internal-api based on where does the request come and what domain is requested. I don't really know how does pow http://pow.cx/ does it but it just works. But pow does not work under linux. 

I want to redirect, or basically serve one application under api.example and app.example, but server totally different one under example.

Adding optional config like
```
{
  redirect: some.domain
}
```
Would really rock! ;)

**tl; dr**
app.example and api.example is same application, but example (without subdomain) it totally different one.

**Comment by typicode** [comment]
@sztosz I was expecting that it wouldn't be so easy :) thank you for the explanations.

This week, I'll be working on fixing a few issues with the UI and add colors to the output. After that I'll have a look at redirections (I have a few ideas already but it will need some refactor).

As a temporary workaround, you could have `www.example.dev` (`hotel add -n www.example ...`) pointing to application A and `*.example.dev`for `app.example.dev` and `api.example.dev` (`hotel add -n example ...`) pointing to application B.

**Comment by sztosz** [comment]
I'll wait and see for now. I can always rename the configuration jsons, because in developement i almost never need to have both app and api up :) but redirections, are really nice to have feature :)
